api/capi/doc/nnstreamer_capi.md

```diff
@@ -0,0 +1,315 @@
+# nnstreamer capi
+
+
+An application which require a data stream pipeline with deep learning model filter can be constructed and assembled easily with nnstreamer c-api.
+
+The main features of nnstreamer-capi include:
+
+- **Construction of data pipeline based on [gstreamer](https://gstreamer.freedesktop.org/)**
+
+  You can compose the data stream pipeline through nnstreamer-capi with various elements which included at gstreamer.
+
+- **[Single-shot](#singleshotapi) & [Pipeline](#pipelineapi) API**
+  
+  There are 2 types of nnstreamer-capi. Single-shot API is useful to get the output result of the model file without the composition of the data pipeline.
+  Otherwise, Pipeline API allows developers to manipulate and compose pipelines with various [elements](#elementapi).
+  
+- **Support various Neural net frameworks (NNFW)**
+  
+  Tensorflow and Tensorflow-Lite are supported. It means the model files, trained through Tensorflow or Tensorflow-lite, can be imported as a filter(element) of a pipeline.
+  Moreover, the custom filter, trained in any kind of way, can be applied as a shared library.
+
+## Prerequisite <a name="prerequisite"></a>
+
+To use the APIs, below headers and features are required.
+
+- **Header**
+  ```bash
+  <nnstreamer.h>
+  <nnstreamer-single.h>
```

Comment:
Uh.. if you use Pipeline API, you don't need nnstreamer-single.h

Reply:
it's moved to Single API section